sshdconfig/src/get.rs

```diff
@@ -9,20 +9,23 @@ use {
 };
 
 use rust_i18n::t;
+use serde_json::{Map, Value};
 use tracing::debug;
 
 use crate::args::Setting;
 use crate::error::SshdConfigError;
+use crate::export::invoke_export_to_map;
+use crate::util::{extract_metadata_from_input, extract_sshd_defaults, SshdCommandArgs};
 
 /// Invoke the get command.
 ///
 /// # Errors
 ///
 /// This function will return an error if the desired settings cannot be retrieved.
-pub fn invoke_get(setting: &Setting) -> Result<(), SshdConfigError> {
+pub fn invoke_get(input: Option<&String>, setting: &Setting) -> Result<(), SshdConfigError> {
```

Comment:
My preference would be that these helper functions return the result of the operation and the `main` code writes it to STDOUT so ideally any writing of output is in a single place.